vsock_sample/py/vsock-sample.py

```diff
@@ -0,0 +1,79 @@
+# Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+#!/usr/local/bin/env python3
+import argparse
+import socket
+import sys
+
+# Client
+class VsockStream:
+    def __init__(self, conn_tmo=5):
+        self.conn_tmo = conn_tmo
+
+    # Connect to the remote endpoint
```

Comment:
Documentation in python is done by adding a docstring inside the function (or class):
```python
def myrand():
    """myrand returns a randomly generated number"""
   return 4
```